docs/source/pages/Frequently-asked-questions-(FAQ).rst

```diff
@@ -0,0 +1,70 @@
+.. role:: raw-html-m2r(raw)
+   :format: html
+
+
+Frequently asked questions
```

Comment:
We should name it `Frequently Asked Questions (FAQ)`

Reply:
Yes